### PR TITLE
[FA] First draft of Chapter4/Page1

### PR DESCRIPTION
**Pull request by hamedonline**
- added chapter4/page1
- updated _toctree.yml to include added page

Related to https://github.com/huggingface/course/issues/50

Reviews are welcome @jowharshamshiri @schoobani

### CODE REVIEW

File: chapters/fa/chapter4/1.mdx
Comment (jowharshamshiri) [inline chapters/fa/chapter4/1.mdx:10]: I propose a replacement for this sentence, which if I didn't have the original English, Honestly I would not understand. Also I don't understand the footnote. "Hosted Inference API" is not a single concept, nor will it be repeated to point to the same deployment, nor is it really impossible for Persian to express. The current choice for deploy in the glossary is "be kargiri". I think it has a half-space after "be" which is my bad, will u be please kind enough to also update that entry? I agree that @schoobani 's choice of "esteghrar" was cool.
See T/point10 on indefinite articles in Persian. Also "bar rooye" I think is redundant.
Can we go with "علاوه بر این، اشتراک گذاری مدل‌ روی هاب به طور خودکار باعث استقرار API برای اجرای آن‌ در هاب می‌شود."

Comment (jowharshamshiri) [reply chapters/fa/chapter4/1.mdx:10]: I checked the style guide, I could not find any rules for "har kas" but it repeatedly uses the similar "hargah" which is a single word therein with no space in btw.

File: chapters/fa/chapter4/1.mdx
Comment (jowharshamshiri) [inline chapters/fa/chapter4/1.mdx:18]: Is the equivalent for hub "paygah" or "haab"? I say that bc even if this is not a technical word, you are introducing a loan word and then refusing to use it, which confuses the reader. Also it is a brand for Hugging Face, so we want to keep it consistent bc of that too. My opinion? Your choice of "paygah" works best.

File: chapters/fa/_toctree.yml
Comment (jowharshamshiri) [inline chapters/fa/_toctree.yml:18]: I did a quick search for your equivalent of tokenizer, "token-saz" on Google, which is what a newbie reading this word in the course will do, all the content there relates to bitcoin wallets and whatnot.--this is the Persian information bubble we talked about before. Sad side effect of this is also breaking the automatic link btw Tokenizer-related classes and methods in code, which is a privilege you are denying our Persian-speaking readers. I think we should stick with transliteration on this one.

**Review by jowharshamshiri — CHANGES_REQUESTED** [review]
Please include updates to the glossary page.

File: chapters/fa/_toctree.yml
Comment (hamedonline) [reply chapters/fa/_toctree.yml:18]: Oh sorry, this is my bad; I forgot to apply what we agreed on discussion page, fixed it now.

File: chapters/fa/chapter4/1.mdx
Comment (hamedonline) [reply chapters/fa/chapter4/1.mdx:4]: > پلت‌فرمی As a loanword I don't think we need to break this into two parts. But interesting case, bc we previously transliterated Hugging Face as هاگینگ‌فِیس (with a half-space) bc in English it is two words but in Persian it is one semantic unit(a brand), here the original English is also one word so my original point stands. Please add this as an update to the glossary in your PR, we don't have it there.

Ok, this got me thinking. What's your opinion on using "سکوی نرم‌افزار" for "platform"?

File: chapters/fa/chapter4/1.mdx
Comment (hamedonline) [reply chapters/fa/chapter4/1.mdx:2]: done

File: chapters/fa/chapter4/1.mdx
Comment (jowharshamshiri) [reply chapters/fa/chapter4/1.mdx:4]: I love your original choice of transliterating platform, first-class Persian word we here all the time. I just think we shouldn't break it into two words in Persian transliteration.

Comment (hamedonline) [reply chapters/fa/chapter4/1.mdx:4]: > Please add dataset to the glossary, new word also.

done.

Comment (jowharshamshiri) [reply chapters/fa/chapter4/1.mdx:4]: Yes I agree, good call.

File: chapters/fa/chapter4/1.mdx
Comment (hamedonline) [reply chapters/fa/chapter4/1.mdx:6]: > I previously mistook pardazesh-goftar for pardazesh-zaban-tabieei. Forgive me. I still think we should keep to the original acronym here tho. Please see the glossary and TRANSLATING/point2. This is an industry-accepted acronym.

I totally agree with your point here. Yet, I believe we've got to consider an exception for the second "NLP" occurrence on this page. I'm afraid a rendering view inconsistency arises when we use a Persian comma "،" right after an English word (being NLP here) in the whole sentence. See the attached screenshot.
Please let me know what you think about the right approach for the second one, I already modified the first occurrence to "NLP". 

render view

Comment (jowharshamshiri) [reply chapters/fa/chapter4/1.mdx:6]: I think that might be an issue with your editor? But since the second sentence already has pardazeshe-goftar and pardazeshe-tasvir in it, I think your choice of pardazeshe-zaban-tabiei in the sentence totally works and sounds better. :)

File: chapters/fa/chapter4/1.mdx
Comment (hamedonline) [reply chapters/fa/chapter4/1.mdx:8]: > No one will ever say مخزن for repository in real life. Can we transliterate? Also I think maybe the preposition here should be در instead of از?

Yes, you're spot on for proposition. On the repository thing, please check this out and maybe we'll decide afterwards:
http://4xmen.ir/ترجمه-صحیح-تمامی-اصطلاحات-گیت-git-به-زبان/

Comment (jowharshamshiri) [reply chapters/fa/chapter4/1.mdx:8]: I just checked the link, I think the repeated use of "Parsi" instead of "Farsi" as the endonym for Persian, disqualifies anything else that site has to say on translation. But then I did a google search and saw how there are a ton of articles referring to makhzan-e-geet. So, I agree let's go with makhzan.

Comment (hamedonline) [reply chapters/fa/chapter4/1.mdx:8]: > Please add entries for versioning and reproducibility to the G. I also don't see the need for footnotes here. Also for reproducibility, can we go with "tekrar-paziri"? "baz-tolid" I think is not self-explanatory.

Sure, I'll add them. I totally agree with you on "reproducibility" translation, "تکرارپذیری" feels better; Probably even better in this form: قابلیت تکرارپذیری
On the footnotes matter: this is a pretty common practice that masters / PhD students follow in their Persian Thesis writings. I personally believe it helps clarify things, rather than being an overkill. It's your call, just let me know.

Comment (hamedonline) [reply chapters/fa/chapter4/1.mdx:8]: > I'm thinking we should transliterate Git, bc it's a word the Persian-speaking industry uses everyday, but the Persian transliteration will clash with Gate, what do you think? Maybe we could make an exception for this one and use the English. We should generally avoid that tho.
> 
> Second thought, I think we should go ahead and transliterate, people reading this have software context in mind. No confusion with equivalent of gate.

The "gate" resemblance is exactly what I thought and made me keep it as "Git". I did some googling now, and looks like both "مخزن" and "گیت" are common as version tracking terms, especially when used together. Check out the attached screenshot.
git_repository

Comment (jowharshamshiri) [reply chapters/fa/chapter4/1.mdx:8]: > > Please add entries for versioning and reproducibility to the G. I also don't see the need for footnotes here. Also for reproducibility, can we go with "tekrar-paziri"? "baz-tolid" I think is not self-explanatory.
> 
> 
> 
> Sure, I'll add them. I totally agree with you on "reproducibility" translation, "تکرارپذیری" feels better; Probably even better in this form: قابلیت تکرارپذیری
> 
> On the footnotes matter: this is a pretty common practice that masters / PhD students follow in their Persian Thesis writings. I personally believe it helps clarify things, rather than being an overkill. It's your call, just let me know.

You know what I think the problem with footnotes?

1. If you're a newbie, and you see a word like tekrar-paziri with a footnote to an unfamiliar and in this case big English word like REPRODUCIBILITY, you get the idea that this is something you won't understand and there is more to it than the simple idea at its core which Persian can sufficiently express.

2. It's like the author did not believe Persian was enough of a language to express this simple concept fully without needing to resort to linking into English lexicon.

3. Some of these pages are really long, like mine in chapter 2. I get how people would do that in papers and books but here it might as well be an endnote bc it is totally obscured from view when encountering the word and prompts scrolling out of your focus zone only to come across a practically useless English equivalent, which we already have in the glossary--the endnote of all endnotes.

4. There might be cases where the English footnote is of actual use, but VERSIONING and REPRODUCIBILITY hardly seem like it. Like, when transliterating the name of a Person it makes total sense to have footnotes with the English spelling, bc we won't have that name in the glossary and that is information that is lost if we don't have a footnote.

Comment (hamedonline) [reply chapters/fa/chapter4/1.mdx:8]: I'll go ahead with the convention the guide itself uses for "گذاری" postfix (by utilizing half-space on all occasions). This will be the final form: به اشتراک‌گذاری
Regarding "هم‌رسانی", it seems to me it looks more like "همدلی" (which is also exemplified in the guide), rather than "تورم‌زا". So, I'll go with "همرسانی".
On that note, should we also include these for "share / sharing" inside G?
gozaari

Comment (jowharshamshiri) [reply chapters/fa/chapter4/1.mdx:8]: Agreed on both. Also, yes lets include share with both these equivalents and their correct spellings in the glossary.

Comment (hamedonline) [reply chapters/fa/chapter4/1.mdx:8]: Fixed.

File: chapters/fa/chapter4/1.mdx
Comment (hamedonline) [reply chapters/fa/chapter4/1.mdx:10]: > I propose a replacement for this sentence, which if I didn't have the original English, Honestly I would not understand. Also I don't understand the footnote. "Hosted Inference API" is not a single concept, nor will it be repeated to point to the same deployment, nor is it really impossible for Persian to express. The current choice for deploy in the glossary is "be kargiri". I think it has a half-space after "be" which is my bad, will u be please kind enough to also update that entry? I agree that @schoobani 's choice of "esteghrar" was cool. See T/point10 on indefinite articles in Persian. Also "bar rooye" I think is redundant. Can we go with "علاوه بر این، اشتراک گذاری مدل‌ روی هاب به طور خودکار باعث استقرار API برای اجرای آن‌ در هاب می‌شود."

Thanks for the suggestion. This is the new translation based on your proposal:
علاوه بر این، اشتراک‌گذاری مدل‌ روی هاب، به طور خودکار باعث استقرار API برای اجرای آن‌ مدل می‌شود.
Regarding the more proper translation of "Deploy / Deployment", I'm also in favor of "استقرار / مستقر کردن". Therefore we can update the glossary with these replacements and forget about "بکارگیری". Or we'll have to start a fresh argument about which form is preferred, "به‌کارگیری" or "به کارگیری"; just like the "به اشتراک‌گذاری" case :))

Comment (jowharshamshiri) [reply chapters/fa/chapter4/1.mdx:10]: :)) Agreed on all points and we didn't have an argument did we. 🥱😴💤

Comment (hamedonline) [reply chapters/fa/chapter4/1.mdx:10]: > I checked the style guide, I could not find any rules for "har kas" but it repeatedly uses the similar "hargah" which is a single word therein with no space in btw.

Well, we're officially in hair-splitting phase. TBH, "هر کس" looks a little more authentic to me.

File: chapters/fa/chapter4/1.mdx
Comment (hamedonline) [reply chapters/fa/chapter4/1.mdx:18]: Again, this is my bad, sorry for inconsistent translation. Forgot to apply the change here.
For the "website" equivalent, is "وب‌سایت" with half-space cool with you?

File: chapters/fa/chapter4/1.mdx
Comment (hamedonline) [reply chapters/fa/chapter4/1.mdx:8]: > Agreed on both. Also, yes lets include share with both these equivalents and their correct spellings in the glossary.
> 
> Really need to sleep now Hamed Jan, you go ahead I'll be with you again in the morning. Apologies in advance for late replies.

❤️

File: chapters/fa/chapter4/1.mdx
Comment (hamedonline) [reply chapters/fa/chapter4/1.mdx:10]: > :)) Agreed on all points and we didn't have an argument did we. 🥱😴💤

Of course we didn't, why should we? :D
❤️❤️❤️

Comment (jowharshamshiri) [reply chapters/fa/chapter4/1.mdx:10]: ❤️❤️❤️

Comment (jowharshamshiri) [reply chapters/fa/chapter4/1.mdx:10]: > > I checked the style guide, I could not find any rules for "har kas" but it repeatedly uses the similar "hargah" which is a single word therein with no space in btw.
> 
> 
> 
> Well, we're officially in hair-splitting phase. TBH, "هر کس" looks a little more authentic to me.

Believe me that's not what I'm trying to do here. Sorry if it looked like that on your end, was not my intention. Look at this in the context of all my other comments, I was knee-deep in the style guide by the time I got to this. :)) I respect your choice.

File: chapters/fa/chapter4/1.mdx
Comment (jowharshamshiri) [reply chapters/fa/chapter4/1.mdx:18]: Yes, that'd be great. I can't see the changes. I think you might've forgotten to push upstream to your fork.

File: chapters/fa/chapter4/1.mdx
Comment (hamedonline) [reply chapters/fa/chapter4/1.mdx:10]: > > > I checked the style guide, I could not find any rules for "har kas" but it repeatedly uses the similar "hargah" which is a single word therein with no space in btw.
> > 
> > 
> > Well, we're officially in hair-splitting phase. TBH, "هر کس" looks a little more authentic to me.
> 
> Believe me that's not what I'm trying to do here. Sorry if it looked like that on your end, was not my intention. Look at this in the context of all my other comments, I was knee-deep in the style guide by the time I got to this. :)) I respect your choice.

You may not believe but I'm actually enjoying your attention to details. I absolutely appreciate what you're doing here and frankly can't thank enough. It sets the stage for a top-notch collaborative work ❤️ .

File: chapters/fa/chapter4/1.mdx
Comment (hamedonline) [reply chapters/fa/chapter4/1.mdx:18]: > Yes, that'd be great. I can't see the changes. I think you might've forgotten to push upstream to your fork.

I've made the changes on local clone of the fork so far, hopefully will push them back to fork and upstream after double checking.

File: chapters/fa/chapter4/1.mdx
Comment (jowharshamshiri) [reply chapters/fa/chapter4/1.mdx:10]: Thank you, very kind of you. ❤️  I too enjoy our collaboration.

**Comment by jowharshamshiri** [comment]
@hamedonline supercool and thanks for the diacritics fix in _toctree.yml. I think you forgot to include the glossary file, please push that too.

**Comment by hamedonline** [comment]
> @hamedonline supercool and thanks for the diacritics fix in _toctree.yml. I think you forgot to include the glossary file, please push that too.

I thought of a separate push for that in which we could discuss about some other suggestions & tweaks I had in mind. Do you agree?

**Comment by hamedonline** [comment]
> > @hamedonline supercool and thanks for the diacritics fix in _toctree.yml. I think you forgot to include the glossary file, please push that too.
> 
> I thought of a separate push for that in which we could discuss about some other suggestions & tweaks I had in mind. Do you agree?

Ok, never mind. I'll commit along and we'll have additional discussions on that file 😃.

**Comment by jowharshamshiri** [comment]
> > @hamedonline supercool and thanks for the diacritics fix in _toctree.yml. I think you forgot to include the glossary file, please push that too.
> 
> I thought of a separate push for that in which we could discuss about some other suggestions & tweaks I had in mind. Do you agree?

We need a second draft PR on this page, I think you missed a couple of small spots but don't want to drag this review too long. By making small commits on each review point and checking them together we can avoid this.

As for the glossary, please include your changes in this PR. I can fetch the changes to my fork so that I can retroactively apply them to my pages in my next PR. I am also available if you want to discuss these changes and anything else you have in mind. If you need other collaborators we can mention them here and ask for their opinion. This ensures everybody always has the latest glossary when working(T/point5). With frequent fetching they can clearly see what changed and apply the changes, which minimizes the editing required and shortens the review process.

**Comment by hamedonline** [comment]
> We need a second draft PR on this page, I think you missed a couple of small spots but don't want to drag this review too long. By making small commits on each review point and checking them together we can avoid this.

@jowharshamshiri I'm super-ok to continue resolving them here, please open open up review remarks on missed spots.

File: chapters/fa/glossary/1.mdx
Comment (jowharshamshiri) [inline chapters/fa/glossary/1.mdx:73]: I don't think this should be one word. Check here https://fa.wikipedia.org/wiki/%D8%AE%D8%B7_%D8%AA%D9%88%D9%84%DB%8C%D8%AF

File: chapters/fa/glossary/1.mdx
Comment (jowharshamshiri) [inline chapters/fa/glossary/1.mdx:105]: Super cool use of diacritics. Is the dash necessary? The original English word does not dictate here.

File: chapters/fa/glossary/1.mdx
Comment (jowharshamshiri) [inline chapters/fa/glossary/1.mdx:106]: Same here.

**Review by jowharshamshiri — CHANGES_REQUESTED** [review]
Thanks so much for this. I'd love to hear your choice for "attention mechanism" I need that in my page.

File: chapters/fa/glossary/1.mdx
Comment (hamedonline) [reply chapters/fa/glossary/1.mdx:73]: Totally agree. I just modified it because of the half-space in the same title in _toctree.yml file. We should modify that file as well to remove the half-space then.

**Review by jowharshamshiri — CHANGES_REQUESTED** [review]

File: chapters/fa/glossary/1.mdx
Comment (hamedonline) [reply chapters/fa/glossary/1.mdx:73]: > I don't think this should be one word. Check here https://fa.wikipedia.org/wiki/%D8%AE%D8%B7_%D8%AA%D9%88%D9%84%DB%8C%D8%AF

Totally agree. I just modified it because of the half-space in the same title in _toctree.yml file. We should modify that file as well to remove the half-space then.

File: chapters/fa/glossary/1.mdx
Comment (hamedonline) [reply chapters/fa/glossary/1.mdx:105]: > Super cool use of diacritics. Is the dash necessary? The original English word does not dictate here.

Let's put them all in one frame and vote. Which one do you think reads easier for an end-user? @jowharshamshiri @schoobani

- بُعدبالا
- بُعد بالا
- بُعد-بالا

File: chapters/fa/glossary/1.mdx
Comment (hamedonline) [reply chapters/fa/glossary/1.mdx:106]: Same voting here @jowharshamshiri @schoobani:

- چندین‌بُعدی
- چندین بُعدی
- چندین-بُعدی
- چندبُعدی
- چند بُعدی
- چند-بُعدی

File: chapters/fa/glossary/1.mdx
Comment (jowharshamshiri) [reply chapters/fa/glossary/1.mdx:105]: 2 I think.

**Comment by hamedonline** [comment]
>

File: chapters/fa/glossary/1.mdx
Comment (jowharshamshiri) [reply chapters/fa/glossary/1.mdx:106]: I vote 2. Actually can't decide btw 2 and 5, vote for both.

**Comment by hamedonline** [comment]
> Thanks so much for this. I'd love to hear your choice for "attention mechanism" I need that in my page.

@jowharshamshiri  Here are the nominees I can think of (I did some googling too):

- مکانیزم توجه
- سازوکار توجه
- مکانیزم ژرف‌نگری

The first one though seems to be more common in the literature.

**Comment by jowharshamshiri** [comment]
> 

If you are ok with it, I like the first one too.

File: chapters/fa/chapter4/1.mdx
Comment (hamedonline) [reply chapters/fa/chapter4/1.mdx:8]: > یک مخزن گیت T/point10 I suggest dropping "yek", the sentence works without it.

fixed, thanks.

File: chapters/fa/glossary/1.mdx
Comment (hamedonline) [inline chapters/fa/glossary/1.mdx:44]: @jowharshamshiri Do you think adding diacritics here is an overkill?

- کد -> کُد

File: chapters/fa/glossary/1.mdx
Comment (hamedonline) [inline chapters/fa/glossary/1.mdx:68]: I have two more suggestions for "Self-Contained" translation:

- "خودکفا"
- "مستقل"

Please share opinions on this @jowharshamshiri @schoobani.

Comment (jowharshamshiri) [reply chapters/fa/glossary/1.mdx:68]: I love 1.

**Comment by hamedonline** [comment]
@jowharshamshiri I have couple of more suggestions for glossary terms, perhaps not necessary for this PR; we can talk about them later on issue page.

- Subword -> زیرکلمه
- Index, as in an array or list -> اندیس
- Tokenized -> توکِن‌شده

**Comment by jowharshamshiri** [comment]
> @jowharshamshiri I have couple of more suggestions for glossary terms, perhaps not necessary for this PR; we can talk about them later on issue page.
> 
> 
> 
> - Subword -> زیرکلمه
> 
> - Index, as in an array or list -> اندیس
> 
> - Tokenized -> توکِن‌شده

All are cool with me.

File: chapters/fa/glossary/1.mdx
Comment (jowharshamshiri) [reply chapters/fa/glossary/1.mdx:105]: Hamed jan, @schoobani jan, check out these links. They all say the same thing. Sorry if there's so many the google search was filled with Arabic results wanted to save you some time. I only filtered for Persian results:

- https://andisheyeamari.irstat.ir/article-1-814-fa.html
- https://en.civilica.com/doc/383285/
- https://www.magiran.com/paper/1894974
- https://shahroodut.ac.ir/fa/thesis/thesis.php?thid=QA547
- https://www.virascience.com/thesis/788224/
- https://www.aparat.com/v/DWLv1/%D8%A2%D9%85%D8%A7%D8%B1_%D8%A8%D8%B9%D8%AF_%D8%A8%D8%A7%D9%84%D8%A7_-_%D9%85%D8%A7%D8%B1%D8%AA%DB%8C%D9%86_%D9%88%DB%8C%D9%86%D8%B1%D8%A7%DB%8C%D8%AA_-_%D9%81%D8%B5%D9%84_%D9%BE%D9%86%D8%AC%D9%85%D8%8C_%D9%82%D8%B3%D9%85%D8%AA
- http://ijece.saminatech.ir/ar/Article/28291
- https://unitahlil.ir/product/%D8%A7%D9%86%D8%AA%D8%AE%D8%A7%D8%A8-%D9%85%D8%AA%D8%BA%DB%8C%D8%B1-%D9%88-%D8%AA%D8%B4%D8%AE%DB%8C%D8%B5-%D8%B3%D8%A7%D8%AE%D8%AA%D8%A7%D8%B1-%D8%AF%D8%B1-%D8%A8%D8%B9%D8%AF-%D8%A8%D8%A7%D9%84%D8%A7/
- http://math.ipm.ac.ir/

**Comment by lewtun** [comment]
Wow, thanks for this contribution @hamedonline and the incredible review from @jowharshamshiri 😍 ! Please let me know when you think this is good to be merged :)

File: chapters/fa/glossary/1.mdx
Comment (hamedonline) [reply chapters/fa/glossary/1.mdx:105]: > Hamed jan, @schoobani jan, check out these links. They all say the same thing. Sorry if there's so many the google search was filled with Arabic results wanted to save you some time. I only filtered for Persian results:
> 
> * https://andisheyeamari.irstat.ir/article-1-814-fa.html
> * https://en.civilica.com/doc/383285/
> * https://www.magiran.com/paper/1894974
> * https://shahroodut.ac.ir/fa/thesis/thesis.php?thid=QA547
> * https://www.virascience.com/thesis/788224/
> * https://www.aparat.com/v/DWLv1/%D8%A2%D9%85%D8%A7%D8%B1_%D8%A8%D8%B9%D8%AF_%D8%A8%D8%A7%D9%84%D8%A7_-_%D9%85%D8%A7%D8%B1%D8%AA%DB%8C%D9%86_%D9%88%DB%8C%D9%86%D8%B1%D8%A7%DB%8C%D8%AA_-_%D9%81%D8%B5%D9%84_%D9%BE%D9%86%D8%AC%D9%85%D8%8C_%D9%82%D8%B3%D9%85%D8%AA
> * http://ijece.saminatech.ir/ar/Article/28291
> * https://unitahlil.ir/product/%D8%A7%D9%86%D8%AA%D8%AE%D8%A7%D8%A8-%D9%85%D8%AA%D8%BA%DB%8C%D8%B1-%D9%88-%D8%AA%D8%B4%D8%AE%DB%8C%D8%B5-%D8%B3%D8%A7%D8%AE%D8%AA%D8%A7%D8%B1-%D8%AF%D8%B1-%D8%A8%D8%B9%D8%AF-%D8%A8%D8%A7%D9%84%D8%A7/
> * http://math.ipm.ac.ir/

IMHO, the thing is we'd better take measures that help differentiate these two concepts (which there are examples for both in your links):

- high dimension: بُعدِ بالا
- high-dimensional: بُعدبالا

**Comment by jowharshamshiri** [comment]
> Wow, thanks for this contribution @hamedonline and the incredible review from @jowharshamshiri heart_eyes ! Please let me know when you think this is good to be merged :)

Just waiting on that vote tally @hamedonline jan. If you think we have enough to go on, please remove the dashes in the glossary update for the two words and replace with full spaces. If you do that, also for the second word please choose 2 or 5 which will match the first one. Other than that LGTM!

File: chapters/fa/glossary/1.mdx
Comment (hamedonline) [reply chapters/fa/glossary/1.mdx:106]: > I vote 2. Actually can't decide btw 2 and 5, vote for both.

My vote goes to 5. We'll take it for now then.

**Comment by hamedonline** [comment]
> > Wow, thanks for this contribution @hamedonline and the incredible review from @jowharshamshiri heart_eyes ! Please let me know when you think this is good to be merged :)
> 
> Just waiting on that vote tally @hamedonline jan. If you think we have enough to go on, please remove the dashes in the glossary update for the two words and replace with full spaces. If you do that, also for the second word please choose 2 or 5 which will match the first one. Other than that LGTM! ❤️❤️❤️

It's done and I believe we're ready to merge @lewtun ❤️.

**Review by lewtun — APPROVED** [review]
Amazing, thank you for translating these sections! Will merge once the CI passes :)

**Pull request merged**
